cmd/list.go

```diff
@@ -248,12 +245,12 @@ func displayListPayload(
 				if humanReadable {
 					value = workflow.Size.HumanReadable
 				} else {
-					value = workflow.Size.Raw
+					value = int(workflow.Size.Raw)
```

Comment:
This cast was needed because the library doesn't support `int64`. Any ideas of improvements would be welcome